main.yml

```diff
@@ -669,12 +708,28 @@ user-preferences:
   top_level: true
   frontend:
     title: User preferences
+    module:
+      appName: user-preferences
+      scope: userPreferences
+      module: ./RootApp
+      group: email
     paths:
       - /user-preferences
     sub_apps:
       - id: email
         title: Email preferences
         default: true
+        group: email
+      - id: notifications
+        title: Notifications
+        group: email
+        sub_apps:
```

Comment:
After removing the `manifest` (locally) it's complaining about this one:
```
Error in user-preferences app definition.
 [Error [ValidationError]: "frontend.sub_apps[1].sub_apps" is not allowed] {
  _original: {
    title: 'User Preferences',
    deployment_repo: 'https://github.com/RedHatInsights/user-preferences-frontend-build',
    top_level: true,
    frontend: {
      title: 'User preferences',
      module: [Object],
      paths: [Array],
      sub_apps: [Array]
    },
    source_repo: 'https://github.com/RedHatInsights/user-preferences-frontend'
  },
  details: [
    {
      message: '"frontend.sub_apps[1].sub_apps" is not allowed',
      path: [Array],
      type: 'object.unknown',
      context: [Object]
    }
  ]
}
```

Reply:
Yeah, the validator is not up to date on stable branch.